src/syrupy/__init__.py

```diff
@@ -39,11 +40,12 @@ def pytest_assertrepr_compare(op: str, left: Any, right: Any) -> Optional[List[s
     Return explanation for comparisons in failing assert expressions.
     https://docs.pytest.org/en/latest/reference.html#_pytest.hookspec.pytest_assertrepr_compare
     """
+    snapshot_text = gettext("snapshot")
     if isinstance(left, SnapshotAssertion):
-        assert_msg = f"snapshot {op} {right}"
+        assert_msg = f"{snapshot_text} {op} {right}"
```

Comment:
This should be the same as the fixture name. Doesn't make sense to translate.

Reply:
Updated to allow custom names